packages/sdk/python/human-protocol-sdk/test/human_protocol_sdk/test_job.py

```diff
@@ -387,22 +373,6 @@ def test_job_bulk_payout_with_full_qualified_url(self):
 
             transaction_retry_mock.reset_mock()
 
-            # Bulk payout with option to store final results publicly
```

Comment:
Why was this deleted?

Reply:
We have removed the option store_pub_final_results=True, so this test is exactly the same as the previous one. In addition, it expects to store the file publicly, but with the new module everything we store is publicly